src/modules/simulator/simulator.h

```diff
@@ -211,7 +211,8 @@ class Simulator {
 	_baro_pub(nullptr),
 	_gyro_pub(nullptr),
 	_mag_pub(nullptr),
-	_initialized(false)
+	_initialized(false),
+	_vehicle_status_updated(false)
```

Comment:
Could you use the timestamp instead and initialise vehicle status like this: _vehicle_status{}, which will set the timestamp to zero.